### DISCUSSION TITLE
Change variant based on breakpoint

### DISCUSSION THREAD

**Discussion by clarkewing** [livewire/flux#914] [Feature Requests]
My main use case for this would be the modal component. It'd be nice to have it use a bottom flyout on small screens and a modal on larger desktop displays.

**Comment by kylewallace** [comment]
Would you want the content of the modal to change based on the breakpoint as well?

**Reply by clarkewing** [reply]
> Would you want the content of the modal to change based on the breakpoint as well?

No, the goal here would simply be to change the look of the modal depending on the user's device.

**Comment by timgavin** [comment]
I was just going to add a new discussion with the same. I'm creating a component for a flux modal and I'm setting the variant and position via MobileDetect, but it would be so much better to do this with tailwind breakpoints instead.
